### PR TITLE
Add test for minion default channel

### PR DESCRIPTION
**Pull request by mbologna**
A minion should always have a default base channel associated.
https://github.com/SUSE/spacewalk/pull/715

### CODE REVIEW

File: features/salt_minion_details.feature [hunk 1]
```diff
@@ -15,3 +18,10 @@ Feature: Verify the minion registration
     And I follow "List / Remove"
     Then I should see a "aaa_base" text
     And I should see a "aaa_base-extras" text
+
+  Scenario: Accepted minion have a base channel
```
Comment (moio) [inline features/salt_minion_details.feature]: Grammar nitpick: "have" is plural :smiling_imp:

**Comment by mbologna** [comment]
@mcalmer @moio @mseidl this BDD test added works by assuming that mgr-sync has populated the related products.
Obviously, if database does not contain any product, minion should not have an associated channel.

This brings me to the question: do we want to sophisticate more the test to check if database products are added or is not necessary?

**Comment by moio** [comment]
We execute tests in order, so we control when to run this test (before or after `mgr-sync refresh`).

To me this test is sufficient - it does not make too much sense to ensure it does not work before `mgr-sync refresh`. I would leave negative testing for jUnit.

**Comment by MalloZup** [comment]
the test "have" a nice style,  for me :)

**Pull request merged**